### PR TITLE
Add monitor and milestones to boba

### PR DESCRIPTION
**Pull request by luke7211**

### CODE REVIEW

**Comment by antooni** [comment]
There are some contracts listed on L2Beat that are missing in discovered.json:

https://etherscan.io/address/0xC891F466e53f40603250837282eAE4e22aD5b088
https://etherscan.io/address/0x1A26ef6575B7BBB864d984D9255C069F6c361a14
https://etherscan.io/address/0x5fD2CF99586B9D92f56CbaD0A3Ea4DF256A0070B

**Review by michalsidzej — APPROVED** [review]

**Review by michalsidzej — CHANGES_REQUESTED** [review]
Some changes that we can see in the discovery should be updated in the config
L1MultiMessageRelayer is missing now
L1LiquidityPool, L1NFTBridge changed address

**Comment by luke7211** [comment]
> Some changes that we can see in the discovery should be updated in the config L1MultiMessageRelayer is missing now L1LiquidityPool, L1NFTBridge changed address

Missing contracts added

**Review by antooni — CHANGES_REQUESTED** [review]
Milestones:
- Boba launches L2 on BNB
- Boba launches L2 on Avalanche
- Boba launches L2 on Moonbeam and Fantom

do not  qualify as milestones because it seems like a regular deployment on EVM compatible chain, correct me if I am wrong

**Review by michalsidzej — APPROVED** [review]

**Review by michalsidzej — APPROVED** [review]
Please respond to Antoni's request. Otherwise LGTM

File: packages/config/src/layer2s/bobanetwork.ts [hunk 1]
```diff
@@ -327,6 +328,31 @@ export const bobanetwork: Layer2 = {
     },
   ],
   milestones: [
+    {
+      name: 'Boba launches L2 on BNB',
```
[the remark this answers is not in the document]
Comment (luke7211) [reply packages/config/src/layer2s/bobanetwork.ts:334]: This is, for Boba, a significant milestone given their strategy. We are not tracking Boba on other chains, however this milestone explains removal a significant chunk of BOBA tokens from Ethereum Rollup

**Review by antooni — APPROVED** [review]

**Pull request merged**